foundry/applications/formApplications/playlistConfig.d.ts

```diff
@@ -1,9 +1,32 @@
 /**
  * Playlist Configuration Sheet
- *
- * @param object - The Playlist being edited
- * @param options - Additional application rendering options
  */
-declare class PlaylistConfig extends FormApplication {
-  _updateObject(event?: Event, formData?: object): Promise<any>; // TODO: Add correct return type
+declare class PlaylistConfig extends BaseEntitySheet<PlaylistConfig.Data, Playlist> {
+  /**
+   * @defaultValue
+   * ```typescript
+   * const options = super.defaultOptions;
+   * options.id = "playlist-config";
+   * options.template = "templates/playlist/edit-playlist.html";
+   * options.width = 360;
+   * ```
+   */
+  static get defaultOptions(): BaseEntitySheet.Options;
+
+  /**
+   * @override
+   */
+  get title(): string;
+
+  /**
+   * @param options - (unused)
+   * @override
+   */
+  getData(options?: Application.RenderOptions): PlaylistConfig.Data;
+}
+
+declare namespace PlaylistConfig {
+  interface Data extends Duplicated<PlaylistConfig['object']['data']> {
```

Comment:
its a bit lucky that this works. You can only extend interfaces and in theory, `Duplicated` might not be typed as an interface. So I would prefer to write this as `type Data = Duplicated<PlaylistConfig['object']['data']> & { ... };`

Reply:
When I tried to change this to a plain `type`, I ran into a lot of problems with circular references and other references breaking.

Reply:
ok, I guess as long as it works, it's fine. I'm a bit confused as to why that would happen though...